src/main/java/werkit/WerkIt.java

```diff
@@ -30,6 +35,8 @@
     private DayList dayList;
     private static Logger logger = Logger.getLogger(WerkIt.class.getName());
 
+    private static final String CASCADE_KEYWORD = "/delete";
```

Comment:
Since the cascade keyword is a 'delete' action keyword, would it be more feasible to use an existing constant such as `WorkoutCommand.DELETE_ACTION_KEYWORD`? This way some redundancy can be removed (in my opinion, at least).

Reply:
Will do that, thank you